frontend/lib/package.json

```diff
@@ -41,6 +41,7 @@
     "@loaders.gl/core": "^3.2.13",
     "@loaders.gl/csv": "^3.2.13",
     "@loaders.gl/gltf": "^3.2.13",
+    "@stlite/kernel": "*",
```

Comment:
do we need this here?

Reply:
Yes, `@stlite/kernel` is used in `@streamlit/lib` such as https://github.com/whitphx/streamlit/blob/dd82e9ba35ff1129fae5d9b7041b4a46dad605ed/frontend/lib/src/components/elements/Audio/Audio.tsx#L20

Reply:
Oki, thats what I thought 👍  I was just wondering since it was missing from your state.

Reply:
Oh I misunderstood the point, my bad, and thanks for the change. You're right.